recipes/pulseaudio/all/conanfile.py

```diff
@@ -122,12 +122,35 @@ def package(self):
         tools.remove_files_by_mask(self.package_folder, "*.la")
 
     def package_info(self):
-        self.cpp_info.libdirs = ["lib", os.path.join("lib", "pulseaudio")]
-        if self.options.with_glib:
-            self.cpp_info.libs.append("pulse-mainloop-glib")
-        self.cpp_info.libs.extend(["pulse-simple", "pulse"])
+        self.cpp_info.components["pulse"].names["pkg_config"] = "libpulse"
+        self.cpp_info.components["pulse"].libs = ["pulse"]
         if not self.options.shared:
-            self.cpp_info.libs.append("pulsecommon-%s" % self.version)
-        self.cpp_info.defines = ["_REENTRANT"]
-        self.cpp_info.names["pkg_config"] = "libpulse"
+            self.cpp_info.components["pulse"].libs.append("pulsecommon-%s" % self.version)
+        self.cpp_info.components["pulse"].libdirs = ["lib", os.path.join("lib", "pulseaudio")]
+        self.cpp_info.components["pulse"].requires = ["libsndfile::libsndfile", "libcap::libcap"]
+        if self.options.with_alsa:
+            self.cpp_info.components["pulse"].requires.append("libalsa::libalsa")
+        if self.options.with_glib:
+            self.cpp_info.components["pulse"].requires.append("glib::glib")
+        if self.options.get_safe("with_fftw"):
+            self.cpp_info.components["pulse"].requires.append("fftw::fftw")
+        if self.options.with_x11:
+            self.cpp_info.components["pulse"].requires.append("xorg::xorg")
+        if self.options.with_openssl:
+            self.cpp_info.components["pulse"].requires.append("openssl::openssl")
+        if self.options.with_dbus:
+            self.cpp_info.components["pulse"].requires.append("dbus::dbus")
+        
+        self.cpp_info.components["pulse-mainloop-glib"].names["pkg_config"] = "libpulse-mainloop-glib"
+        self.cpp_info.components["pulse-mainloop-glib"].defines.append("_REENTRANT")
+        self.cpp_info.components["pulse-mainloop-glib"].libs = ["pulse-mainloop-glib"]
```

Comment:
Previously, `pulse-mainloop-glib` was only added to libs when `self.options.with_glib` was True.